share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -303,14 +303,15 @@ correct. Add an explicit [`version`](http://www.rubydoc.info/github/Homebrew/bre
 
 ## Commit
 
-Everything is built on Git, so contribution is easy:
-
+Everything is built on Git, so contribution is easy:  
+1. Fork the Homebrew/homebrew-core repository on GitHub. This creates a pushable, personal remote repository. This is needed as only Homebrew maintainers have push access to the main repository.  
```

Comment:
Can we retain the "go to the `homebrew-core` repository and hit the fork button](https://github.com/Homebrew/homebrew-core)" language that talks about *how* to do the fork? And the "This is needed..." sentence seems unnecessary, since that's the normal reason for GitHub forks in general.